From 682f1acc490abb3dea60a26e8260b58cd275131b Mon Sep 17 00:00:00 2001
From: klzgrad <kizdiv@gmail.com>
Date: Sun, 16 May 2021 20:40:16 +0800
Subject: [PATCH] build: Disable build_with_chromium

The argument build_with_chromium mainly enables various tests,
data bundling, infra integration, and AFDO profiles.

AFDO can be added by other arguments.
---
 src/build/config/gclient_args.gni | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/build/config/gclient_args.gni b/src/build/config/gclient_args.gni
index 82e4222b8..4cf0124dd 100644
--- a/src/build/config/gclient_args.gni
+++ b/src/build/config/gclient_args.gni
@@ -1,6 +1,6 @@
 # Generated from 'DEPS'
-android_ndk_version = "2@30.0.16138531"
-build_with_chromium = true
+android_ndk_version = "24"
+build_with_chromium = false
 checkout_android = false
 checkout_android_prebuilts_build_tools = false
 checkout_clang_coverage_tools = false
@@ -15,4 +15,4 @@ checkout_src_internal_infra = false
 checkout_clusterfuzz_data = false
 cros_boards = ""
 cros_boards_with_qemu_images = ""
-generate_location_tags = true
\ No newline at end of file
+generate_location_tags = true
-- 
2.53.0

